src/tables/extensions/memoryTable.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import useSWR from "swr";
+import { openmrsFetch } from "@openmrs/esm-framework";
+
+import { InfoDataTable } from "./dataTable";
+
+const MemoryTable: React.FC = () => {
+  const url = `/ws/rest/v1/systeminformation`;
+  const { data, error, isValidating } = useSWR<
+    { data: { systemInformation } },
+    Error
+  >(url, openmrsFetch);
+
+  const isLoading = !data && !error;
+
+  const headerData = [
```

Comment:
These table files seem to be very similar. I would look at making the component more reusable, or even its functions and constants, like `headerData` here which we already said can live outside of the main component

Reply:
The data is going to be different, I created a reusable component for the tables that will receive rows and headers as props, these five tables will have different headers and rowData.